app/src/features/bottomBar/bottomBar.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from 'styled-components';
+import HomeOn from '../../common/images/bottomHomeOn.png';
+import Home from '../../common/images/bottomHome.png';
+import Reference from '../../common/images/bottomReference.png';
+import ReferenceOn from '../../common/images/bottoemReferenceOn.png';
+import MySurvey from '../../common/images/bottomMySurvey.png';
+import User from '../../common/images/bottomUser.png';
+import Plus from '../../common/images/plusButton.png';
+import { theme } from '../../common/styles/theme';
+
+interface Props {
+  path: string;
+}
+
+const BottomBar: React.FC<Props> = ({ path }) => {
+  return (
+    <Wrap>
+      <Button>
+        {path === 'home' ? <Img src={HomeOn} /> : <Img src={Home} />}홈
+      </Button>
+      <Button>
+        {path === 'reference' ? (
+          <Img src={ReferenceOn} />
+        ) : (
+          <Img src={Reference} />
+        )}
+        자료실
+      </Button>
+      <Button>
+        <PlusImg src={Plus} />
+      </Button>
+      <Button>
+        {path === 'mySurvey' ? <Img src={HomeOn} /> : <Img src={MySurvey} />}
+        내설문
+      </Button>
+      <Button>
+        {path === 'User' ? <Img src={HomeOn} /> : <Img src={User} />}마이
+      </Button>
+    </Wrap>
+  );
+};
+
+export default BottomBar;
+
+const Wrap = styled.footer`
+  border-top: 1px solid #f1f1f1;
+  height: 84px;
+  display: flex;
+  justify-content: space-around;
+`;
+
+const Button = styled.button`
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  font-weight: ${theme.typography.medium};
```

Comment:
theme.typography 에 저장된 값이 css 이므로 속성추가 없이 사용해도 될것 같아요
```suggestion
${({theme}) => theme.typography.medium};
```